api/src/canny/canny.handlers.ts

```diff
@@ -81,6 +81,27 @@ export const cannyIdentify = async (
           curriculum: user.curriculum,
         },
       });
+
+      // Update user in PostHog
+      POSTHOG_CLIENT.identify({
+        distinctId: netId,
+        properties: {
+          email: user.email || '',
+          firstName: user.first_name || '',
+          lastName: user.last_name || '',
+          upi: user.upi || -1,
+          school: user.school || '',
+          college: user.college || '',
+          major: user.major || '',
+          curriculum: user.curriculum || '',
```

Comment:
Not sure, but you should be able to omit the default strings.

Reply:
Yeah I gave that a shot, but PostHog doesn't seem to like `undefined` or `null` values for these